docker/iroh-fedimintd/.env

```diff
@@ -0,0 +1,19 @@
+# Bitcoin auth information
+# generate with https://jlopp.github.io/bitcoin-core-rpc-auth-generator/,
+# default is user: bitcoin, pass: bitcoin
+BITCOIND_RPC_AUTH=bitcoin:54ae356e13a76dc8068e960eb43193cb$efeeb347a1f0b4a7b7832cc26e68861bc46f89126a8e136ff9932cad47739041
+
+# This domain should point to the machine fedimintd is being deployed to
+FM_DOMAIN=my-super-host.com
+
+# Where bitcoind is reachable
+# Disabled for iroh demo
+# FM_BITCOIN_RPC_KIND=bitcoind
+# FM_BITCOIN_RPC_URL=http://bitcoin:bitcoin@bitcoin:8332
+
+# For testing or fallback one can also use esplora
+FM_BITCOIN_RPC_KIND=esplora
+FM_BITCOIN_RPC_URL=https://blockstream.info/api/
```

Comment:
todo: mutiny

Reply:
verified setting up a federation with mutinynet and completed a deposit/withdrawal on-chain